local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -57,5 +57,9 @@ module.exports = [
     command: 'verbose',
     description: 'Enables logging',
     default: false,
+  }, {
+    command: 'reset-cache',
+    description: 'Removes cached files',
+    default: false
```

Comment:
@davidaurelio @mroswald should the default be true for bundle command or atleast add this to `react-native-xcode.sh`?